packages/core/src/controllers/SiweController.ts

```diff
@@ -0,0 +1,87 @@
+import { subscribeKey as subKey } from 'valtio/utils'
+import { proxy, subscribe as sub } from 'valtio/vanilla'
+import type {
+  CreateSIWEMessageArgs,
+  SIWESession,
+  VerifySIWEMessageArgs
+} from '../utils/TypeUtil.js'
+
+import { SIWEStatus } from '../utils/TypeUtil.js'
```

Comment:
Second TypeUtil import in the same file, you can combine with previous one. This should've been caught by eslint / prettier though, are they set up properly?

Reply:
`SIWEStatus` is an enum so can't be imported as a type.
I duplicated the import for consistency with current coding styles but I could have a single import like so:
Should I keep it as is or use the snippet below ?

```js
import {
  type SIWECreateMessageArgs,
  type SIWESession,
  type SIWEVerifyMessageArgs,
  SIWEStatus
} from '../utils/TypeUtil.js'
```